### PR TITLE
Created traditions.py and completed the easter_tradition command

### PR DESCRIPTION
**Pull request by RohanRadia**
User runs the command `<prefix>easter_tradition` and it will output a random country's name along side the tradition they they have for Easter!

### CODE REVIEW

File: traditions.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+traditions = {
```
Comment (sco1) [inline traditions.json]: This isn't a valid JSON document. You can remove `traditions = ` to fix this.

Take a look at some of our other [resource JSON files](https://github.com/python-discord/seasonalbot/tree/master/bot/resources) for some more examples.

File: traditions.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "Germany": "In Germany, Easter is known by the name of Ostern. Easter holidays for children last for about three weeks. Good Friday, Easter Saturday and Easter Sunday are the days when people do not work at all.",
+    "Mexico": "Semana Santa and Pascua (two separate observances) form a part of Easter celebrations in Mexico. Semana Santa stands for the entire Holy Week, from Palm Sunday to Easter Saturday, whereas the Pascua is the observance of the period from the Resurrection Sunday to the following Saturday.",
+    "Poland": "They shape the Easter Butter Lamg (Baranek Wielkanocyny) from a chunk of butter. They attempt to make it look like a fluffy sheep!",
+    "Greece": "They burn an effigy of Judas Iscariot, they betrayer of Jesus, sometimes is done as part of a Passion Play! It is hung by the neck and then burnt.",
+    "Philippines": "Some Christians put themselves through the same pain that Christ endured, they have someone naile them to a cross and put a crown of thornes on their head.",
```
Comment (sco1) [inline traditions.json]: Along with the above comment, remove this trailing comma to fix the JSON formatting.

File: bot/seasons/easter/traditions.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                      'Guatemala': 'In Guatemala, Easter customs include a large, colourful celebration marked by countless processions. The main roads are closed, and the sound of music rings through the streets. Special food is prepared such as curtido (a diced vegetable mix which is cooked in vinegar to achieve a sour taste), fish, eggs, chickpeas, fruit mix, pumpkin, pacaya palm and spondias fruit (a Spanish version of a plum.)',
+                      'Germany': 'In Germany, Easter is known by the name of Ostern. Easter holidays for children last for about three weeks. Good Friday, Easter Saturday and Easter Sunday are the days when people do not work at all.',
+                      'Mexico': 'Semana Santa and Pascua (two separate observances) form a part of Easter celebrations in Mexico. Semana Santa stands for the entire Holy Week, from Palm Sunday to Easter Saturday, whereas the Pascua is the observance of the period from the Resurrection Sunday to the following Saturday.',
+                      'Poland': 'They shape the Easter Butter Lamg (Baranek Wielkanocyny) from a chunk of butter. They attempt to make it look like a fluffy sheep!',
```
Comment (odCat) [inline bot/seasons/easter/traditions.py]: Typo: Correct: Lamb

File: bot/seasons/easter/traditions.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                      'Greece': 'They burn an effigy of Judas Iscariot, they betrayer of Jesus, sometimes is done as part of a Passion Play! It is hung by the neck and then burnt.',
```
Comment (odCat) [inline bot/seasons/easter/traditions.py]: Typo: Correct: the betrayer

File: bot/seasons/easter/traditions.py (hunk 3 above)
Comment (RohanRadia) [reply bot/seasons/easter/traditions.py]: Corrected. Thanks. Please re-review.

File: bot/seasons/easter/traditions.py (hunk 4 above)
Comment (RohanRadia) [reply bot/seasons/easter/traditions.py]: Fixed

File: bot/seasons/easter/traditions.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="easter_tradition")
+    async def easter_tradition(self, ctx):
+        """Responds with a random tradition or custom"""
+
+        traditions = {}
+        with open('userbalance.json', 'r') as f:
```
[the remark this answers is not in the document]
Comment (RohanRadia) [reply bot/seasons/easter/traditions.py]: Attempted to fix, not sure if correct. Will commit in a minute.

File: bot/seasons/easter/traditions.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+log = logging.getLogger(__name__)
+
+
+class Traditions(commands.Cog):
+    """A cog which allows users to get a random easter tradition or custom from a random country."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="easter_tradition")
```
[the remark this answers is not in the document]
Comment (RohanRadia) [reply bot/seasons/easter/traditions.py]: Done

File: traditions.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (RohanRadia) [reply traditions.json]: Completed.

Comment (RohanRadia) [reply traditions.json]: Fixed, was a copy and paste problem.

File: traditions.json (hunk 2 above)
Comment (RohanRadia) [reply traditions.json]: Minor mistake. Fixed.

File: bot/seasons/easter/traditions.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    traditions = json.load(f)
+
+
+class Traditions(commands.Cog):
+    """A cog which allows users to get a random easter tradition or custom from a random country."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(alias='eastercustoms')
```
Comment (sco1) [inline bot/seasons/easter/traditions.py]: The kwarg here is [`aliases`](https://discordpy.readthedocs.io/en/rewrite/ext/commands/api.html#discord.ext.commands.Command.aliases), and is a list/tuple:

```suggestion
    @commands.command(aliases=('eastercustoms',))
```

Comment (RohanRadia) [reply bot/seasons/easter/traditions.py]: Committed directly from here.

**Pull request merged**